### PR TITLE
Add Waren Gonzaga Language Quote

### PR DESCRIPTION
**Pull request by WarenGonzaga**
## ✅ New Quote Checklist
- [x] The quote(s) added are in alphabetical order and not ❌ duplicates.
- [x] The quote(s) added are about **learning a foreign language**.
- [x] The sentence attribute does not have inner quotation marks, and is ended with a period.
- [x] Copy pasted the raw JSON `quotes.json` file to [JSONLint](https://jsonlint.com/) and made sure that the modified JSON file is valid.

### CODE REVIEW

**Comment by Kvaibhav01** [comment]
@jayehernandez please take a look at the added quotes.

**Comment by WarenGonzaga** [comment]
> @jayehernandez please take a look at the added quotes.

I guess she is busy she is friend of mine...

**Comment by WarenGonzaga** [comment]
@jayehernandez Just fix the conflict issue...

File: server/data/quotes.json [hunk 1]
```diff
@@ -571,6 +571,14 @@
     "author": "Vivien Leigh",
     "sentence": "I never found accents difficult, after learning languages."
   },
+  {
+    "author": "Waren Gonzaga",
+    "sentence": "Learning foreign languages is like learning computer programming languages."
+  },
+  {
+    "author": "Waren Gonzaga",
+    "sentence": "The more language you know, the better."
```
Comment (jayehernandez) [inline server/data/quotes.json:580]: @WarenGonzaga good with the first quote, not sure about this one though?

**Review by jayehernandez — CHANGES_REQUESTED** [review]

File: server/data/quotes.json (hunk 1 above)
Comment (WarenGonzaga) [reply server/data/quotes.json:580]: @jayehernandez basically and indirectly duplicate?

**Comment by WarenGonzaga** [comment]
@jayehernandez done removing the second quote.

**Comment by jayehernandez** [comment]
Thanks @WarenGonzaga! merging this now :)

**Pull request merged**